### DISCUSSION TITLE
Multiple Shell support

### DISCUSSION THREAD

**Discussion by fazeelbh** [Caliburn-Micro/Caliburn.Micro#869] [Q&A]
Hi,
I have to develop a WPF application with two windows, main window in primary monitor are other other one in secondary monitor. Uses shall be able to work on each windows independently, but while closing the main window child window also need to be closed. If I understand correctly , it requires two shells and conductors.  Is there any way to create multiple shells using Calibur.Micro to achieve this scenario?

Thanks in advance

**Comment by vb2ae** [comment]
what platform are you using for the app?  .net framework or .net 6 or later

**Reply by fazeelbh** [reply]
.Net framework 4.7.2

**Reply by fazeelbh** [reply]
Hi, 
It would be greatly appreciated if you could provide guidance on how to accomplish this

**Reply by Yinimi** [reply]
I am not sure what the problem is. You could just open another window, couldn't you?
